client/__tests__/e2e/e2eAnnotations.test.js

```diff
@@ -85,11 +85,11 @@ describe.each([
   test("create a label", async () => {
     await utils.hoverOn("cluster-test:see-actions");
     await utils.clickOn("cluster-test:add-new-label-to-category");
-    await utils.typeInto("cluster-test:new-label-name", "test-label-name");
+    await utils.typeInto("cluster-test:new-label-name", "test-label");
     await utils.clickOn("cluster-test:submit-label");
     await cxgActions.expandCategory("cluster-test");
-    const result = await utils.waitByID("categorical-value-cluster-test-test-label-name");
-    expect(await result.evaluate(node => node.innerText)).toBe("test-label-name");
```

Comment:
Adjusting this because the length of the label name caused it to be condensed in the sidebar and the equality test assertion to fail.